explorer/app/hooks/useFetchEntity.tsx

```diff
@@ -24,7 +24,7 @@ export const useFetchEntity = <T,>(
 ): UseEntityDetailResponse<T> => {
   const entity = useCurrentEntity();
   const router = useRouter();
-  const uuid = router.query.uuid as string;
+  const uuid = router.query.uuid?.[0] as string;
```

Comment:
Two things:

1. What's at index 0?
1. Can you create a constant for 0?

Reply:
Same reason as the above.
But for this file I want to get the uuid from the `...param` dynamic route `/explore/[slug]/[param] -> /explore/projects/123456/overview` so I need to get the first index.